R/connect_inbo_dbase.R

```diff
@@ -6,11 +6,13 @@
 #' The function can only be used from within the INBO network.
 #'
 #' For more information, refer to
-#' \href{https://inbo.github.io/tutorials/tutorials/r_database_access/}{this tutorial}.
+#' \href{https://inbo.github.io/tutorials/tutorials/r_database_access/}{this
+#' tutorial}.
 #'
 #' @param database_name char Name of the INBO database you want to connect
 #' @param autoconvert_utf8 Should the encoding of the tables that are retrieved
-#' from the database be adapted to ensure correct presentation?  Defaults to TRUE.
+#' from the database be adapted to ensure correct presentation?  Defaults to
```

Comment:
I prefer to start every sentence on a new line. Then changing one sentence will not influence other sentences as they are on different line. No need to change this now. I tend to do this only when I update documentation.